Cargo.toml

```diff
@@ -37,3 +36,6 @@ winapi = "~0.2"
 bench = false
 name = "crust_peer"
 path = "examples/crust_peer.rs"
+
+[features]
+cargo-clippy = []
```

Comment:
The feature is needed to only disable `unknown_lints` in clippy mode. If `unknown_lints` isn't disabled, build will fail in `clippy` mode due to ` +#![cfg_attr(feature="cargo-clippy", allow(too_many_arguments))] `, which isn't a "native Rust lint".

We can only remove this if we change crust signature of some functions: http://pastebin.com/P9H6D1N1